src/commands/shared.ts

```diff
@@ -1,5 +1,16 @@
 import { Argv } from "yargs";
 
+export enum Network {
+  "mainnet" = 1,
+  "ropsten" = 3,
+  "rinkeby" = 4,
+  "goerli" = 5,
+  "polygon" = 137,
+  "polygon_mumbai" = 80001,
```

Comment:
I think we may have do it this way instead:

```suggestion
  "matic" = 137,
  "maticmum" = 80001,
```

The problem here is with our cli implementation using the default strings of ethers directly and use them directly. We might want to refactor how the cli works when handling the network in future so that we free ourselves from the default names if we ever want to go beyond what ethers has.

Reply:
looks like i can't deploy maticmum document store out of the box. As per what u mentioned, is it due to ethersjs [Signer](https://github.com/Open-Attestation/document-store/blob/master/src/index.ts#L3), which the defaults does not support maticmum? So in other to "gain" other networks, we need to pass in a custom provider that uses either infura / alchemy yeah?

Reply:
> looks like i can't deploy maticmum document store out of the box.

Did you mean before or after the changes you have in this PR?

I would expect we should be able to do so with `maticmum` with the changes in this PR. But for any other networks not included in ethers, yea, will need to use a provider for that network.